src/rev_ai/models/customer_url_data.py

```diff
@@ -0,0 +1,19 @@
+class CustomerUrlData:
+    """Object used to contain a url provided by the customer along with any other related information,
+     such as authentication headers."""
+
+    def __init__(self, url, auth_headers=None):
+        """Constructor
+        :param url: customer provided url
+        :param auth_headers: optional customer provided headers to access the url
```

Comment:
I think this need to include type information. Is it a string, an array of strings, or a special auth header object?

Reply:
Added a description and example